tilelang/autotuner/tuner.py

```
@@ -424,17 +435,367 @@ def _load_result_from_disk(self, key) -> AutotuneResult:
+                started = time.perf_counter()
+                latency, worker_ref_latency = benchmark_target(
+                    jit_kernel=jit_kernel,
+                    benchmark_state=worker_state,
+                )
+                elapsed = time.perf_counter() - started
+                if timeout > 0 and elapsed > timeout:
+                    raise TimeoutException("Benchmark timed out")
+                result_queue.put((idx, config, jit_kernel, latency, worker_ref_latency, None, ""))
+            except TimeoutException:
+                result_queue.put((idx, config, jit_kernel, None, None, "timeout", ""))
+            except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)